docker/docker-compose.varnish.yml

```diff
@@ -1,10 +1,10 @@
 ---
-version: "2"
+version: "2.1"
```

Comment:
Oh, good catch! It may be needed in the other docker-compose file for PMM too, so it would be great if you change it in there too.